lib/new_relic/agent/llm/response_headers.rb

```diff
@@ -0,0 +1,28 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic
+  module Agent
+    module Llm
+      module ResponseHeaders
```

Comment:
Since we don't necessarily need to generate a ResponseHeaders event, I thought making this a module would be a good way to keep the unique attributes while also allowing unique methods to be defined to set values.